app/views/shifts/_form.html.erb

```diff
@@ -0,0 +1,42 @@
+<%= form_with(model: shift, local: true) do |form| %>
+  <% if shift.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(shift.errors.count, "error") %> prohibited this shift from being saved:</h2>
+
+      <ul>
+        <% shift.errors.full_messages.each do |message| %>
+          <li><%= message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field">
```

Comment:
This is not a blocker for this PR, but the UI representation is quite unclear right now (`status` being `false/true` as opposed to `open/filled` 😅 

Let's change that later.